src/components/HomePage/SearchArea/AdvancedSearch/FieldOptions.js

```diff
@@ -1,9 +1,9 @@
 /* istanbul ignore file */
 
-export default Object.freeze([
-    { label: "Back-End", value: "BACK_END" },
-    { label: "Front-End", value: "FRONT_END" },
-    { label: "Dev-Ops", value: "DEVOPS" },
-    { label: "Machine Learning", value: "ML" },
-    { label: "Computer Vision", value: "COMPUTER_VISION" },
-]);
+export default Object.freeze({
+    "BACKEND": "Back-End",
+    "FRONTEND": "Front-End",
+    "DEVOPS": "Dev-Ops",
+    "ML": "Machine Learning",
```

Comment:
To match the backend, some keys should be changed. Here should be: "MACHINE LEARNING".